appservice/src/SiteClient.ts

```diff
@@ -231,4 +231,8 @@ export class SiteClient {
             }
         }
     }
+
+    public async getPublishingUser(): Promise<User> {
+        return await this._client.getPublishingUser({});
```

Comment:
Why are you passing an empty object? Is there a slot specific method for this?

Reply:
It requires an options object, all options are optional.  This returns the publishing User associated to Deployment Options (like in the screen shot) rather than the app specific username.

Reply:
Oh, and no, there is no slot option as this is a `client` function, not `client.webapp`